mmv1/third_party/terraform/services/container/resource_container_node_pool_test.go.tmpl

```diff
@@ -3953,49 +3953,63 @@ resource "google_container_cluster" "cluster" {
   deletion_protection = false
   network             = "%s"
   subnetwork          = "%s"
+  version             = "1.32.0-gke.1448000"
```

Comment:
I think these all need to be [min_master_version](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/container_cluster#min_master_version-1)?

```suggestion
  min_master_version  = "1.32.0-gke.1448000"
```

Reply:
specifically the ones for google_container_cluster - node pool is correct to use [version](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/container_node_pool#version-1).

Reply:
Makes sense!